packages/components/src/Actions/ActionDropdown/ActionDropdown.snapshot.test.js

```diff
@@ -28,10 +28,10 @@ describe('ActionDropdown', () => {
 		};
 
 		// when
-		const wrapper = mount(<ActionDropdown {...props} />).find('ActionDropdown');
+		const wrapper = mount(<ActionDropdown {...props} />);
```

Comment:
The `.find('ActionDropdown')` selection was to bypass the i18n HOC but now the HOC mock return the original component. So we don't have this extra level we wanted to bypass.